src/result_set.rs

```diff
@@ -295,12 +346,17 @@ fn parse_statistics(value: Value) -> RedisGraphResult<Statistics> {
 
 /// A scalar value returned by RedisGraph.
 #[derive(Debug, Clone, PartialEq)]
+#[non_exhaustive]
```

Comment:
Why would you want to skip the totality check here?

Reply:
That's my mistake, it was for when it was WIP